### PR TITLE
added color schema for deploy

### LINKED ISSUE #733: `kusk deploy` does not output errors when watching files
**Describe the bug**
When using `kusk deploy -i <file> -w` any errors are not output to the terminal

**To Reproduce**
```
$ kusk deploy -i openapi.yaml -w
api.gateway.kusk.io/simple-api updated
⏳ watching for API changes in openapi.yaml

$ kusk deploy -i openapi.yaml   
Error: no upstream or mocking configuration found for path GET/
 ```
**Expected behavior**
```
$ kusk deploy -i openapi.yaml -w
api.gateway.kusk.io/simple-api updated
⏳ watching for API changes in openapi.yaml
Error: no upstream or mocking configuration found for path GET/
 ```

### PR DESCRIPTION
**Pull request by jasmingacic**
Signed-off-by: jasmingacic <jasmin.gacic@gmail.com>

This PR add color theme #690  to deploy and fixes #733 

image

image

@donnalux 
I might be off with colors I chose as I can't really tell colors apart. I can say that I used 
* Green for "successfully parsed"
* White for "change detected in"
* Red for ❌  errors 
Please direct me to exact color should be used for the theme

### CODE REVIEW

File: cmd/kusk/cmd/root.go [hunk 1]
```diff
@@ -90,7 +91,10 @@ func Execute() {
 		errors.NewErrorReporter(rootCmd, err).Report()
 	}
 
-	cobra.CheckErr(err)
+	if err != nil {
+		fmt.Println(color.FgRed.Render("❌ ", err.Error()))
```
Comment (mbana) [inline cmd/kusk/cmd/root.go:95]: Don't you want to use `kuskui.PrintError(err.Error())` here?

File: cmd/kusk/cmd/deploy.go [hunk 2]
```diff
@@ -121,13 +124,13 @@ var deployCmd = &cobra.Command{
 					reportError(err)
 					return err
 				}
-				fmt.Printf("api.gateway.kusk.io/%s updated\n", api.Name)
+				kuskui.PrintSuccess(fmt.Sprintf("🎉 api.gateway.kusk.io/%s updated", api.Name))
```
Comment (mbana) [inline cmd/kusk/cmd/deploy.go:127]: Doesn't 🎉 get included in `kuskui.PrintSuccess`?

File: cmd/kusk/cmd/deploy.go [hunk 3]
```diff
@@ -93,6 +93,9 @@ var deployCmd = &cobra.Command{
 			return err
 		}
 
+		kuskui.PrintSuccess(fmt.Sprintf("🎉 successfully parsed %s", file))
+		kuskui.PrintStart(fmt.Sprintf("🚢 initiallizing deployment to fleet %s", envoyFleetName))
```
Comment (mbana) [inline cmd/kusk/cmd/deploy.go:97]: Doesn't 🚢 get included in `kuskui.PrintStart`?

**Review by mbana — APPROVED** [review]
LGTM. I have a few  questions, however.

**Comment by donnalux** [comment]
Need a space between the error icon and the error message.

**Review by mbana — DISMISSED** [review]
LGTM.

**Review by kylehodgetts — APPROVED** [review]

**Comment by jasmingacic** [comment]
image
translated the color scheme to gookit

**Review by donnalux — DISMISSED** [review]

File: cmd/kusk/internal/kuskui/ui.go [hunk 4]
```diff
@@ -0,0 +1,28 @@
+package kuskui
```
Comment (jasmingacic) [inline cmd/kusk/internal/kuskui/ui.go:1]: @kylehodgetts I added this to replace testkube's UI library as we have a little bit different color scheme. Plus are not going to use tables and other elements. Can you take a look at this?

**Review by kylehodgetts — DISMISSED** [review]

**Pull request merged**